platform-coop-toolkit/coop-library-framework
# Make title required (resolves #138)

## Description

Resolves #138.

## Steps to test

1. Try to publish a resource with no title.

**Expected behavior:** Publication is prevented, user is alerted.

## Additional information

Not applicable.

## Related issues

- Resolves #138.
